tox.ini

```diff
@@ -70,7 +70,7 @@ commands=
     sphinx-build -W docs build/docs
 
 [testenv:checkdocs]
-basepython= python3.8
+basepython= python3.9
```

Comment:
Setting `basepython` in the `testenv` here is not necessary.  These should just be removed.  If not specified, it will use whatever version is available, or you can do `py38-checkdocs` or `py39-coverage` to get the combo you want.  Hard-coding in the env makes the env less flexible.